doc/boss_tutorial.md

```diff
@@ -1,40 +1,45 @@
-# PaddlePaddle Elastic Deep Learning
+# PaddlePaddle Tutorial for BOSS Workshop 2018
 
-<img src="../logo/edl.png" width="500">
+<img src="../logo/paddle.png" width="300">
 
 PaddlePaddle (PArallel Distributed Deep LEarning) is an easy-to-use, efficient,
 flexible and scalable deep learning platform, which is originally developed by
 Baidu scientists and engineers for the purpose of applying deep learning to many
 products at Baidu.
 
-PaddlePaddle Elastic Deep Learning (EDL) is a clustering project which leverages PaddlePaddle training jobs to
-be scalable and fault-tolerant. EDL will greatly boost the parallel distributed training jobs and make good use
-of cluster computing power.
-
-EDL is based on the full fault-tolerant feature of PaddlePaddle, it uses a Kubernetes controller to manage
-the cluster training jobs and an auto-scaler to scale the job's computing resources.
+Fluid is the latest version of PaddlePaddle, which describes the "process" of training
```

Comment:
which describes ... => it describes the model for training or inference using the representation of "Program".

Reply:
Done.